src/assets/toolkit/styles/vendor/prism.css

```diff
@@ -24,14 +24,15 @@
 .token.attr-value,
 .token.selector,
 .token.class-name {
-  color: color-mod(var(--color-green) l(+15%));
+  color: color-mod(var(--color-green) l(+17%));
```

Comment:
If this was changed to match the updated button color I wonder if it would storing this value in a variable?

Note: I'm missing a _ton_ of context here so I don't know if this suggestion makes sense given our compilation setup, buttons styles, colors, etc.

Reply:
Because the code styles are one of the few areas where we're inverting more than white/gray on dark backgrounds, it's sort of an outlier for the library. You'll in the CSS file that we're tweaking a few colors for this context.

Reply:
Cool, no worries! Like I said I was missing a ton of context, but thought I might be remiss if I didn't mention it at all